### DISCUSSION TITLE
Database tables names

### DISCUSSION THREAD

**Discussion by Ae-Mc** [fastapi-users/fastapi-users#869] [Ideas]
What about table names in real databases?
As I can see, table names for different adapters are different. Maybe it would be better if they all had same names?
Now for each adapter we have different table names:
- sqlalchemy: `user` and `oauth_account`
- ormar: `users` and `oauth_accounts`
- sqlmodel: `user` and `oauthaccount`

In my opinion to simplify switching between adapters tables should  be named equally, but that will break backwards compability.
Or we can allow user select their own names for tables...

**Comment by frankie567** [comment]
Yes, unfortunately, I didn't pay attention to this. It could be nice to be consistent everywhere indeed.

Meanwhile, you can easily override this in SQLAlchemy and SQLModel using `__tablename__` class property:

## SQLAlchemy

```py
class UserTable(Base, SQLAlchemyBaseUserTable):
    __tablename__ = "users"
```

## SQLModel

```py
class UserDB(SQLModelBaseUserDB, table=True):
    __tablename__ = "users"
```

**Comment by Ae-Mc** [comment]
Isn't it important for framework how tables named? Or I can name tables as I want and nothing would break?
